eventhubs/spring-cloud-azure-starter-eventhubs/eventhubs-client/src/main/resources/application.yaml

```diff
@@ -0,0 +1,12 @@
+spring:
+  cloud:
+    azure:
+      eventhubs:
+        namespace: ${AZURE_EVENTHUBS_NAMESPACE}
+        event-hub-name: ${AZURE_EVENTHUB_NAME}
+        consumer:
+          consumer-group: ${AZURE_EVENTHUB_CONSUMER_GROUP}
+        processor:
```

Comment:
this is not correct, producer, consumer, processor maps to three different clients, so if the code uses only producer and consumer clients, then the configurations under processor are not required.